ingestion/ingest_census_data.py

```diff
@@ -0,0 +1,228 @@
+import os
```

Comment:
This looks good. If we want to run this scrip like we've been running the other files `python -m manage runscript <module>` we do need to add a `run()` function that is what is going to be called. If it's possible to add it, that'd be great.

Reply:
@benjaleivas is this file still needed since the `/scripts/` file now exists and can be run the way JP mentioned?

it seems that this is importing `models` and if that is still needed, then this should likely also be migrated to the `/scripts/` folder.